charts/velero/README.md

```diff
@@ -65,7 +69,7 @@ helm install vmware-tanzu/velero --namespace <YOUR NAMESPACE> -f values.yaml --g
 If a value needs to be added or changed, you may do so with the `upgrade` command. An example:
 
 ```bash
-helm upgrade vmware-tanzu/velero <RELEASE NAME> --namespace <YOUR NAMESPACE> --reuse-values --set configuration.provider=<NEW PROVIDER>
+    helm upgrade vmware-tanzu/velero <RELEASE NAME> --namespace <YOUR NAMESPACE> --reuse-values --set configuration.provider=<NEW PROVIDER>
```

Comment:
This change would break the alignment.
![image](https://user-images.githubusercontent.com/49380831/108150969-a6fdd300-7110-11eb-9b9f-199ac841266e.png)

Reply:
Good catch. I'll revert the indent.